utils/bison-c.sh

```diff
@@ -9,7 +9,7 @@ if test "x$BISON" = x -o "x$OUTPUT" = x -o "x$INPUT" = x;  then
   exit 1
 fi
 
-BISON_MAJOR_VER=`${BISON} --version |grep bison|sed -e "s;.* ;;" -e "s;\..*;;"`
+BISON_MAJOR_VER=`"${BISON}" --version |grep bison|sed -e "s;.* ;;" -e "s;\..*;;"`
```

Comment:
Huh? How are these changes related?

Reply:
They're only related in the sense that I couldn't compile, because my path to bison contains spaces. I can move them to a separate PR.